shared-objects/loginData.js

```diff
@@ -2,41 +2,27 @@ const { CLIENT } = require("./servers");
 
```

Comment:
loginData.js should be removed (as is described in ticket SC-5786), move rest data to new  page objects structure (if needed) and delete file (loginData.js)

Reply:
done:)